openedx/core/djangolib/blockstore_cache.py

```diff
@@ -168,7 +176,7 @@ def get_bundle_draft_files_cached(bundle_uuid, draft_name):
     get automatic cache invalidation when the draft is updated.
     """
     bundle_cache = BundleCache(bundle_uuid, draft_name)
-    cache_key = ('bundle_draft_files', )
+    cache_key = ('bundle_draft_files', draft_name)
     result = bundle_cache.get(cache_key)
     if result is None:
         result = list(blockstore_api.get_bundle_files(bundle_uuid, use_draft=draft_name))
```

Comment:
Is it even possible to reach this code anymore? Seems like `bundle_cache.get` will call `get_bundle_version_number()` which will always update this.

Reply:
It doesn't look to be possible in usual circumnstances, but I'd be leery of _not_ having a safety net after a cache get.  After all, it is conceivable that the cache could be cleared in the microseconds between then, and here.